extensions/interactions/TextInput/TextInputValidationService.js

```diff
@@ -17,12 +17,36 @@
  */
 
 oppia.factory('TextInputValidationService', [
-  'baseInteractionValidationService',
-  function(baseInteractionValidationService) {
+  'WARNING_TYPES', 'baseInteractionValidationService',
+  function(WARNING_TYPES, baseInteractionValidationService) {
     return {
       getCustomizationArgsWarnings: function(customizationArgs) {
-        // TODO(juansaba): Implement customization args validations.
-        return [];
+        var warningsList = [];
+        baseInteractionValidationService.requireCustomizationArguments(
+          customizationArgs,
+          ['placeholder', 'rows']);
+
+        var rows = customizationArgs.rows.value;
```

Comment:
Similarly to below, do we have a method for this? I didn't see anything.. not sure if I should hack something together.